learn/how-tos/building-pwa-app.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Enabling PWA - Beta"
+id: "building-pwa-app"
+sidebar_label: "Enabling PWA"
+---
+---
+
+## What is PWA (Progressive Web Application)
+
+PWA (Progressive Web Application) are web applications that provide a native mobile app experience. These applications combine great functionalities and enhanced user experience based on the browser’s capabilities, and works on any platform, including desktop and mobile devices. You can develop PWA using web technologies such as JavaScript, HTML, and CSS.
```

Comment:
Add to this paragraph this sentence:

WaveMaker web applications are already responsive and ready to be converted to Progressive Web App. In this document you will learn how to turn any WaveMaker web application into a PWA.